src/services/httpClient.js

```diff
@@ -67,8 +68,8 @@ export class HttpClient {
     const { retryWaitTime, maxRetryAttempts } = Config;
     const request = HttpClient.createAjaxRequest({ ...options, url }, withAuth);
     return Observable.ajax(request)
-      // If success, extract the response object
-      .map(({ response }) => response)
+      // If success, extract the response object and enforce camelCase keys
+      .map(({ response }) => camelcaseKeys(response, { deep: true }))
```

Comment:
After some more thought, I think I agree that this is just an extra step in parsing the json data which will just slow it down without much benefit.  The dot notation is far more helpful than this.

Reply:
Yeah. Let's leave this in this PR to avoid making too many issues in the next one. I'll make another PR to remove it from this and the rules and actions stuff.